app/models/gem_ownership_transfer.rb

```diff
@@ -0,0 +1,5 @@
+class GemOwnershipTransfer < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :ruby_gem
+  validates :ruby_gem, :old_user_id, :new_user_id, presence: true
```

Comment:
I think it's better to validate the presence of the association than just the foreing key, so maybe something like:
```ruby
validates :ruby_gem, :old_user, :new_user, presence: true
```